src/Microsoft.PowerShell.Security/security/CredentialCommands.cs

```diff
@@ -78,6 +78,19 @@ public string Title
         }
 
         private string _title = UtilsStrings.PromptForCredential_DefaultCaption;
+        
+        /// <summary>
+        /// Gets and sets whether to confirm the password.
+        /// </summary>
+        [Parameter(Mandatory = false)]
+        [ValidateNotNullOrEmpty]
+        public SwitchParameter ConfirmPassword
+        {
+            get { return _confirmPassword; }
+            set { _confirmPassword = value; }
+        }
+
+        private bool _confirmPassword;
```

Comment:
Please use auto-implemented property.
```suggestion
        public SwitchParameter ConfirmPassword { get; set; }
```